encrypted-media-respec.html

```diff
@@ -4037,9 +4049,7 @@
         </section>
         <section>
           <h4>Mitigations</h4>
-          <p>User agents SHOULD present the interfaces for clearing <a def-id="distinctive-identifiers"></a> and Key System stored data in a way that helps users to understand this possibility and enables them to delete data in all persistent identification and storage features, including HTTP session cookies [[!COOKIES]] and web storage, simultaneously.
-            Likewise, User agents SHOULD present the interfaces for disabling and re-enabling a Key System in a way that helps users to understand this possibility and enables them to delete all such data in all persistent storage features simultaneously.
-          </p>
+          <p>Recomendations mitigating these concerns are defined in <a href="#persistent-state-requirements"></a>.</p>
```

Comment:
ack spelling. The referenced section contains only recommendations related to these concerns whereas ir does indeed contain requirements relevant to the previous section.